tests/scripts/all.sh

```diff
@@ -3448,6 +3448,18 @@ component_test_malloc_0_null () {
     tests/ssl-opt.sh -e 'proxy'
 }
 
+component_test_aes_only_128_bit_keys () {
+    msg "build: default config with AES_ONLY_128_BIT_KEY_LENGTH enabled"
+    scripts/config.py set MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH
+    scripts/config.py set MBEDTLS_CTR_DRBG_USE_128_BIT_KEY
```

Comment:
Similar to previous comment from @tom-cosgrove-arm, we should ensure here that `MBEDTLS_CTR_DRBG_C` is not enabled, since it will superceed `MBEDTLS_CTR_DRBG_USE_128_BIT_KEY` leaving the CTR DRBG to use a 256-bit key.